### PR TITLE
add a tools-and-libraries section to the Community page

### PR DESCRIPTION
**Pull request by SethTisue**
(prompted by recent discussions about improving the Scala newbie
experience)

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -63,11 +65,23 @@ See our [events page](/events/).
...
 * [Scala Exercises](http://scala-exercises.47deg.com/)
 * [Scala School](http://twitter.github.com/scala_school/)
 * [Effective Scala](http://twitter.github.com/effectivescala/)
 * [Scala Puzzlers](http://scalapuzzlers.com/)
 * [Interactive Tour](http://scalatutorials.com/tour)
-* [Awesome Scala](https://github.com/lauris/awesome-scala)
+
+## Community Libraries and Tools
+
+finding libraries:
```
Comment (heathermiller) [inline community/index.md]: I know it's a stupid request, but capitalize? Pretty please?

File: community/index.md [hunk 2]
```diff
@@ -63,11 +65,23 @@ See our [events page](/events/).
...
+
+* [Awesome Scala](https://github.com/lauris/awesome-scala) is "a community driven list of useful Scala libraries, frameworks and software"
+* the [Tools and Libraries page](https://wiki.scala-lang.org/display/SW/Tools+and+Libraries) on the (now mostly retired) Scala wiki also lists and categorizes Scala libraries
+* [Typelevel.org](http://typelevel.org) provides an assortment of popular libraries and extensions to Scala.
+
+staying current:
```
Comment (heathermiller) [inline community/index.md]: Here too

File: community/index.md [hunk 3]
```diff
@@ -63,11 +65,23 @@ See our [events page](/events/).
...
 * [Scala Puzzlers](http://scalapuzzlers.com/)
 * [Interactive Tour](http://scalatutorials.com/tour)
-* [Awesome Scala](https://github.com/lauris/awesome-scala)
+
+## Community Libraries and Tools
+
+finding libraries:
+
+* [Awesome Scala](https://github.com/lauris/awesome-scala) is "a community driven list of useful Scala libraries, frameworks and software"
+* the [Tools and Libraries page](https://wiki.scala-lang.org/display/SW/Tools+and+Libraries) on the (now mostly retired) Scala wiki also lists and categorizes Scala libraries
```
Comment (heathermiller) [inline community/index.md]: Here too :)

**Comment by heathermiller** [comment]
LGTM :)

**Comment by dwijnand** [comment]
Perhaps this could link to https://github.com/explore, perhaps links like https://github.com/trending?l=scala&since=weekly and https://github.com/stars?language=scala.

I find the daily email useful/interesting.

**Comment by SethTisue** [comment]
new commits: HeaTHeR's WisH is mY cOmmaND, GitHub link added (good suggestion Dale; I added what seems to me like the single best link)

**Comment by SethTisue** [comment]
merging, but further feedback remains welcome

**Pull request merged**